site/docs/patterns/header-block.mdx

```diff
@@ -0,0 +1,148 @@
+
+<Diagram
+  src="/img/patterns/header-block/header-block-layout-spacing.png"
+  alt="A header block layout."
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Header block with `--salt-spacing-100` padding does not contain an accent bar. We recommend using this header block size within components where visual emphasis is not important. Consider the available space for an accent bar, relative to the size of the component.
+</ExampleContainer>
+
+## Layout variations
+
+A header block has a number of possible layout variations depending on the amount of information you want to communicate, the user workflow and the container it will sit in.
+
+### With accent bar and all text elements
+
+Display headings with an accent bar which draws attention to the heading, reinforces brand identity, and remains consistent with Salt Design System’s theming options.
+
+
+<Diagram
+  src="/img/patterns/header-block/with-accent-bar-and-text.png"
+  alt="Dialog with a header block using padding --salt-spacing-300"
+  content="Dialog with a header block using padding --salt-spacing-300"
+  border
+/>
+
+### With accent bar, heading and pre-header only
+
+The dialog example demonstrates a step in a process as introduced by the pre-header and heading. A description is not necessary as the content of the dialog is sufficiently self-explanatory for the user to navigate and complete the form field instructions.
+
+<Diagram
+  src="/img/patterns/header-block/accent-bar-heading-preheader.png"
+  alt="Dialog with a header block using heading and pre header only."
+  content="Dialog with a header block using heading and pre header only."
+  border
+/>
+
+### With accent bar, heading and description only
+
+The drawer example uses a heading accompanied by a description that provides additional information on the status of the content. A contextual overview of the content is not required in the form of a pre-header, as the content itself indicates a summary of information.
+<Diagram
+  src="/img/patterns/header-block/header-block-in-drawer.png"
+  alt="Drawer with a header block using accent bar, heading and description."
+  border
+/>
+
+### Small container
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/header-block/small-container.png", alt: "" },
+    { src: "/img/patterns/header-block/small-container-spacing.png", alt: "" },
+  ]}
+  label="Show overlay content area"
+  caption="Overlay with a header block using padding --salt-spacing-100"
+/>
+
+### Close button only
+Display a close button if a container element needs to be dismissed via a close button.
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/header-block/close-button-only.png", alt: "" },
+    { src: "/img/patterns/header-block/close-button-only-spacing.png", alt: "" },
+  ]}
+  label="Show overlay content area"
+  caption="Overlay with a header block using padding --salt-spacing-100, and close button on display only"
+/>
+
+### With status
```

Comment:
I got to the end and realised this doesn't feel like it follows any of our previous patterns. This should be a lot more similar to contact details that it currently is.